### DISCUSSION TITLE
键盘上下键移动的时候，如何让表格选中的单元格始终在屏幕上出现

### DISCUSSION THREAD

**Discussion by LeePiaPia** [Happy-Coding-Clans/vue-easytable#522] [General]
比如表格过长的时候，我用上下键移动的时候，如何让选中的单元格，自动滚上来

**Comment by huangshuwei** [comment]
哪个版本。我这看好像没出现你说的问题：https://happy-coding-clans.github.io/vue-easytable/#/zh/doc/table/cell-selection?anchor=dan-xuan-shi-li-fang-fa

**Reply by LeePiaPia** [reply]
你这个是表格固定高度，内部滚动，我遇到的是页面滚动
